jadx-gui/src/main/java/jadx/gui/ui/hexviewer/HexSearchBar.java

```diff
@@ -270,19 +270,29 @@ private void makeFindByHexButton() {
 		findTypeCB.setToolTipText(NLS.str("search.find_type_hex"));
 	}
 
-	private boolean isValidHexSting(String hexSting) {
-		String cleanS = hexSting.replace(" ", "");
+	private boolean isValidHexSting(String hexString) {
```

Comment:
"Sting" typo still present in method name... It's probably fine, I'm not even reviewer so don't mind me...

Reply:
> "Sting" typo still present in method name... It's probably fine, I'm not even reviewer so don't mind me...

thanks